src/util/DataUtils.ts

```diff
@@ -11,6 +11,10 @@ export const mathSign = (value: number) => {
   return -1;
 };
 
+// use legacy isFinite only if there is a problem (aka IE)
+// eslint-disable-next-line no-restricted-globals
+export const isFinit = Number.isFinite ? Number.isFinite : isFinite;
```

Comment:
this is probably fine to remove entirely now

Reply:
Great!